examples/reqwest-report.rs

```diff
@@ -0,0 +1,105 @@
+use threescalers::api_call::*;
+use threescalers::application::*;
+use threescalers::credentials::*;
+use threescalers::extensions::Extensions;
+use threescalers::http::request::FromRequest;
+use threescalers::http::Request;
+use threescalers::service::*;
+use threescalers::transaction::Transaction;
+use threescalers::usage::Usage;
+
+use reqwest::Client;
+use reqwest::{RequestBuilder, Response};
+
+fn main() -> Result<(), threescalers::errors::Error> {
+    let creds = Credentials::ServiceToken(ServiceToken::from("12[3]token"));
+    let svc = Service::new("svc123", creds);
+    let uks = [
+        "userkey_1",
+        "userkey_2",
+        "userkey_3",
+        "userkey 4",
+        "userkey 5",
+    ];
+    let apps = uks
+        .into_iter()
+        .map(|uk| Application::from(UserKey::from(*uk)))
+        .collect::<Vec<_>>();
+
+    println!("Apps: {:#?}", apps);
+
+    let usages = [
+        ("metric11", 11),
+        ("metric12", 12),
+        ("metric21", 21),
+        ("metric22", 22),
+        ("metric31", 31),
+        ("metric32", 32),
+        ("metric41", 41),
+        ("metric42", 42),
+        ("metric51", 51),
+        ("metric52", 52),
+    ]
+    .chunks(2)
+    .map(|metrics_and_values| Usage::new(metrics_and_values))
+    .collect::<Vec<_>>();
+
+    println!("Usages: {:#?}", usages);
+
+    let ts = Default::default();
+
+    let txns = apps
+        .iter()
+        .zip(&usages)
+        .map(|(a, u)| Transaction::new(a, None, Some(u), Some(&ts)))
+        .collect::<Vec<_>>();
+
+    let mut extensions = Extensions::new();
+    extensions.insert("no_body", "1");
+    extensions.insert("testing[=]", "0[=:=]0");
```

Comment:
?

Reply:
this allows you to see how the escaping is done